content/_code-samples/freeze/py/enable_no_freeze.py

```diff
@@ -0,0 +1,27 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountSet, AccountSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnet
+
+
+# generate wallet
+sender_wallet = generate_faucet_wallet(client=client)
+
+# build accountset transaction to disable freezing
+accountset = AccountSet(account=sender_wallet.classic_address, set_flag=AccountSetFlag.ASF_NO_FREEZE)# flag to disable freezing for this account
```

Comment:
The comment goes off the screen which makes it hard to read. Normally I'd recommend putting it in front, or splitting the line up, but in this case I think the flag name is descriptive enough.
```suggestion
accountset = AccountSet(account=sender_wallet.classic_address, set_flag=AccountSetFlag.ASF_NO_FREEZE)
```